### PR TITLE
Feature/jupyter improvements

### PR DESCRIPTION
**Pull request by carlba**
These are some small suggestions to make this more similar to the other jupyter roles I created.

* A jupyter specific environment file is used. It only has the necessary requirements for jupyter.
* The conda environment is called jupyter.
* No o3 folder is created in the jupyter home directory since it is no longer needed.
* I also threw in the fix for the false changed state for the conda updated environment.

### CODE REVIEW

File: roles/anaconda3/tasks/main.yml [hunk 1]
```diff
@@ -79,18 +79,20 @@
...
-    creates: /home/{{ anaconda3_user }}/anaconda3/envs/o3
+    creates: /home/{{ anaconda3_user }}/anaconda3/envs/jupyter
   become: yes
   become_user: "{{ anaconda3_user }}"
   register: conda_environment_created
   tags: anaconda3
 
 - name: ensure conda environment is updated based on environment-linux.yml
-  shell: bash -il -c 'conda env update --name o3 -f {{ anaconda3_environment_file }}'
+  shell: bash -il -c 'conda env update --name jupyter -f {{ anaconda3_environment_file }}'
```
Comment (mblomdahl) [inline roles/anaconda3/tasks/main.yml]: This won't work! De-generalizes the `anaconda3` role and makes "jupyter" envs for everyone.

File: roles/anaconda3/tasks/main.yml [hunk 2]
```diff
@@ -79,18 +79,20 @@
   tags: anaconda3
 
 - name: ensure conda environment is created based on environment-linux.yml
-  shell: bash -il -c 'conda env create --name o3 python=3.6 -f {{ anaconda3_environment_file }}'
+  shell: bash -il -c 'conda env create --name jupyter python=3.6 -f {{ anaconda3_environment_file }}'
```
Comment (mblomdahl) [inline roles/anaconda3/tasks/main.yml]: Same thing here and below.

**Review by mblomdahl — CHANGES_REQUESTED** [review]
Looks like a good idea. But some things that needs to be addressed.

File: roles/anaconda3/tasks/main.yml (hunk 1 above)
Comment (carlba) [reply roles/anaconda3/tasks/main.yml]: Yea absolutely stupid mistake will fix :)

**Pull request closed without merging**